packages/engine/src/FeatureFlagsState.tsx

```diff
@@ -33,7 +33,7 @@ export const FeatureFlagsState = defineState({
   initial: {} as Record<string, boolean>,
   enabled(flagName: string) {
     const state = getMutableState(FeatureFlagsState)[flagName].value
-    return typeof state === 'boolean' ? state : true
+    return typeof state === 'boolean' ? state : false // returns false if state has not been set
```

Comment:
we want to preserve the default functionality of everything being enabled for local dev. we either need to seed everything into the database, or return this to default to true.

Reply:
makes sense for local dev

is there somewhere we are turning flags off for features that we don't want to be enabled in other environments?

Reply:
or should we just go ahead and seed/store key-value pairs for each flag in the db per environment?

noticing that we're reaching a point where there are several parts of the studio that others are asking to hide/disable beyond the local env

Reply:
reverting is good for now, we can address seeding later